test/unit/io/record_test.cpp

```diff
@@ -15,11 +15,23 @@
 #include <seqan3/io/detail/record.hpp>
 #include <seqan3/core/concept/tuple.hpp>
 #include <seqan3/std/algorithm>
+#include <seqan3/test/expect_range_eq.hpp>
 
 using seqan3::operator""_dna4;
 
 using default_fields = seqan3::fields<seqan3::field::seq, seqan3::field::id, seqan3::field::qual>;
 
+// This is needed for EXPECT_RANGE_EQ:
+namespace seqan3
+{
+template <typename char_t>
+inline debug_stream_type<char_t> & operator<<(debug_stream_type<char_t> & stream, field f)
```

Comment:
I think we claim that all our types are "debug streamable" so this should be part of the API. But it can be done in another PR (to answer your next question). Could you please add a card (or how you would prefer to track this)?

Reply:
I'm not sure, but I think not every type needs to be "debug streamable", e.g. the detail::types. I don't know if we had a resolution of what types need this. I think we only had a design goal to make every (public) seqan3 type serialisable.

However you like; Could you please add this card? This is io related and I don't know what you would expect here as an output. I just did some hack here and casted the enum as its value representation. Maybe strategy meeting?

Reply:
Yes, only the public API should be debug-streamable but fields are public. 
We can add this to the DEV meeting although it has low priority I think.